pkg/bot/teams.go

```diff
@@ -173,8 +173,8 @@ func (b *Teams) processActivity(w http.ResponseWriter, req *http.Request) {
 
 	err = b.Adapter.ProcessActivity(ctx, activity, coreActivity.HandlerFuncs{
 		OnMessageFunc: func(turn *coreActivity.TurnContext) (schema.Activity, error) {
-			resp := b.processMessage(turn.Activity)
-			if len(resp) >= maxMessageSize {
+			n, resp := b.processMessage(turn.Activity)
+			if n >= maxMessageSize {
```

Comment:
were you able to test that successfully?
![Screen Shot 2022-09-29 at 11 14 57](https://user-images.githubusercontent.com/17568639/192992362-00eea352-7277-4e4b-a17a-bc256cd5dc3b.png)

Reply:
Will double check this one.

Reply:
It's still not working for me, however the directory was created on the OneDrive side. 
![Screen Shot 2022-09-30 at 11 31 02](https://user-images.githubusercontent.com/17568639/193240339-83110f50-9416-47e1-aa16-59e937bdb682.png)